tests/conformance/configuration/configuration.go

```diff
@@ -203,12 +216,16 @@ func ConformanceTests(t *testing.T, props map[string]string, store configuration
 	}
 
 	if config.HasOperation("subscribe") {
+		subscribeMetadata := make(map[string]string)
+		if component == postgres {
```

Comment:
Ideally Conformance tests should not have component specific code as they test the interface, but subscribe call for postgres required channel in the metadata and so it's added here